bootique-jetty/pom.xml

```diff
@@ -28,6 +28,11 @@
 			<artifactId>jetty-servlet</artifactId>
 			<scope>compile</scope>
 		</dependency>
```

Comment:
We are adding a new dependency to `bootique-jetty` . I suggest creating a separate module for it. Perhaps `bootique-jetty-servlets`, with config prefix "jettyservlets". So cors filter may be configured as

```yaml
jettyservlets:
   cors:
```